vendor/github.com/livepeer/lpms/segmenter/video_segmenter.go

```diff
@@ -252,11 +252,12 @@ func (s *FFMpegVideoSegmenter) pollPlaylist(ctx context.Context, fn string, slee
 }
 
 func (s *FFMpegVideoSegmenter) pollSegment(ctx context.Context, curFn string, nextFn string, sleepTime time.Duration) (f []byte, err error) {
+	var content []byte
```

Comment:
I don't have anything to back this up, but it seemed like golang cleared the unused memory faster when you assign new values to the same variable (vs. creating a new variable)